dependencies/download_torch.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2020-2022 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import argparse
+import glob
+from urllib.request import urlretrieve
+import os
+import warnings
+
+
+def search_on_path(filenames):
+    for p in os.environ.get('PATH', '').split(os.pathsep):
+        for filename in filenames:
+            full = os.path.join(p, filename)
+            if os.path.exists(full):
+                return os.path.abspath(full)
+    return None
+
+
+def get_cuda_path():
+    nvcc_path = search_on_path(('nvcc', 'nvcc.exe'))
+    cuda_path_default = None
+    if nvcc_path is not None:
+        cuda_path_default = os.path.normpath(os.path.join(os.path.dirname(nvcc_path), '..', '..'))
+    if cuda_path_default is not None:
+        _cuda_path = cuda_path_default
+    elif os.path.exists('/usr/local/cuda'):
+        _cuda_path = '/usr/local/cuda'
+    else:
+        _cuda_path = None
+
+    return _cuda_path
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--cuda_path", help="Path to installed cuda", type=str, default=None)
+    parser.add_argument("--opendr_device", help="Target device for installation",
+                        type=str, choices=["gpu", "cpu"], default="gpu")
+    parser.add_argument("--torch_version", help="Version of Libtorch to be installed", type=str, default="1.9.0")
```

Comment:
Because LibTorch and Torch are the same thing I think both are correct but my help message speaks directly about the C - CPP implementation of the library. Maybe is should rephrased better, like:

parser.add_argument("--torch_version", help="Specifies LibTorch version to be installed", type=str, default="1.9.0")`

Reply:
Because LibTorch and Torch are the same thing I think both are correct but my help message speaks directly about the C - CPP implementation of the library. Maybe is should rephrased better, like:

`parser.add_argument("--torch_version", help="Specifies LibTorch version to be installed", type=str, default="1.9.0")`

Reply:
Ok yes, that makes sense, thank you.